[#287] Add Java linting (#295)
airyhq/airy: CONTRIBUTING.md

You can push as many commits we need to finish a feature.

## Lint your changes

> We use a couple of language specific linters. For Bazel code we use [buildifier](https://github.com/bazelbuild/buildtools/tree/master/buildifier),
> for web projects [prettier](https://prettier.io/), and for Java [CheckStyle](https://checkstyle.sourceforge.io/).

Reviewer: ```suggestion
We use language specific linters: [buildifier](https://github.com/bazelbuild/buildtools/tree/master/buildifier) to lint Bazel files, [prettier](https://prettier.io/) for TypeScript, JavaScript and SCSS, [CheckStyle](https://checkstyle.sourceforge.io/) for Java.
```

> Java linting is run as a test, so depending on the package you can run

Reviewer: ```suggestion
Java linting is run as a test, so depending on the package you can run:
```

```sh
bazel test //my/package:checkstyle
```

to have Bazel check your Java code for linting errors.

> Buildifier and Prettier are bundled up in a single Bazel command. So you can run

Reviewer: ```suggestion
Buildifier and Prettier are bundled up in a single Bazel command. So you can run:
```

```sh
bazel run //:check
```

> to check your code and

Reviewer: ```suggestion
to check your code and:
```

```sh
bazel run //:fix
```

> to have the linters try and fix and the issues.

Reviewer: ```suggestion
to try fixing linting issues automatically.
```
